app/javascript/src/components/Dashboard/Tasks/NewTaskForm.jsx

```diff
@@ -0,0 +1,101 @@
+import React from "react";
+import * as yup from "yup";
+import { Formik, Form } from "formik";
+import { Input, Textarea, Select } from "neetoui/formik";
+import { Button, Switch } from "neetoui";
+
+const tagValues = [
+  { value: "Internal", label: "Internal" },
+  { value: "Agile Workflow", label: "Agile Workflow" },
+  { value: "Bug", label: "Bug" },
+];
+
+const contacts = [
+  { value: "Karthik Menon", label: "Karthik Menon" },
+  { value: "MS Dhoni", label: "MS Dhoni" },
+];
+
+export default function NewTaskForm({ onClose }) {
+  const handleSubmit = () => {
+    alert("form submitted");
+  };
+  return (
+    <Formik
+      initialValues={{
+        title: "",
+        description: "",
+        showDueDateField: true,
+      }}
+      onSubmit={handleSubmit}
+      validationSchema={yup.object({
+        title: yup.string().required("Title is required"),
+        description: yup.string().required("Description is required"),
+        tag: yup.object().required("Tag is required"),
+      })}
+    >
+      {({ isSubmitting, values }) => (
+        <Form className="pb-10">
+          <Input label="Title" name="title" className="mb-6" />
+          <Select
+            className="mb-6"
+            label="Tags"
+            defaultValue={null}
+            placeholder="Select a tag"
+            isDisabled={false}
+            isClearable={true}
+            isSearchable={true}
```

Comment:
Can't we just provide `isDisabled`, `isClearable` and `isSearchable` instead of `isDisabled={false}` and so on?